doc/source/whatsnew/v0.19.0.txt

```diff
@@ -24,6 +24,16 @@ New features
 
 .. _whatsnew_0190.dev_api:
 
+to_datetime can be used with Offset
```

Comment:
``to_datetime has gained an origin kwarg``.

don't call this ``Offset`` which is a very specific meaning